app/server/publish.js

```diff
@@ -19,18 +19,19 @@ Meteor.publish('myClasses', function(flags) {
     // like return Class.find(flags);
 });
 
-Meteor.publish('getViewStates', function(section, flags) {
-    /*
-        if(Meteor.user) {
-            //something
-        }
-
-        check (section, String);
-        check (flags, { something });
-        flags = _.assignInWith(flags, { section });
-
-        return View.find(flags);
-    */
+Meteor.publish('getViewStates', function () {
+    var currentUserId = this.userId;
+    var currentUser = Meteor.users.findOne({userId : currentUserId});
```

Comment:
Copy paste the ff to line 23:
Para lang to sa conventions

var currentUserId = this.userId,
    currentUser = Meteor.users.findOne({userId : currentUserId});